Java/core/src/main/java/com/amazon/randomcutforest/IComponentModel.java

```diff
@@ -19,4 +19,5 @@
 import com.amazon.randomcutforest.executor.IUpdatable;
 
 public interface IComponentModel<P> extends ITraversable, IUpdatable<P> {
+    void setBoundingBoxCacheFraction(double fraction);
```

Comment:
I think the other option we could implement for this would be to add a `getTree()` method either to this interface or to `IUpdatable`. Then in the forest and the update executor we add a method:
```
public <R> List<R> mapTrees(Function<ITree<?>, R> function)
```
Then we could implement `setBoundingBoxCacheFraction` in `RandomCutForest` like this:
```
public void setBoundingBoxCacheFraction(double cacheFraction) {
  updateExecutor.mapTrees(t -> t.setBoundingBoxCacheFraction(cacheFraction));
}

Reply:
Done!